internal/cmd/scraper/scraper.go

```diff
@@ -167,11 +167,16 @@ func RunWithEmitters(cfg *Config, emitters []integration.Emitter) error {
 		)
 	}
 
+	fetcher, err := integration.NewFetcher(scrapeDuration, cfg.ScrapeTimeout, cfg.WorkerThreads, cfg.BearerTokenFile, cfg.CaFile, cfg.InsecureSkipVerify, queueLength)
+	if err != nil {
+		logrus.Errorf("Error building fetcher: %v", err)
```

Comment:
we might want to add some more context of the consequences of this error since it has error level and will pop up in the default logs

Reply:
Actually this error should be fatal, I don't know why I logged it rather than throwing it up 🤦🏻‍♀️

Reply:
Done, error is now fatal.
Regarding the context, error should be properly wrapped so at the end message should contain a meaningful message :)